### PR TITLE
GDSC BIT

### PR DESCRIPTION
**Pull request by todcru**

### CODE REVIEW

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@todcru Can you change your picture to `400px*400px`?

Everything else looks good: https://deploy-preview-11--gdscleadmap.netlify.app/

**Comment by todcru** [comment]
Sure

**Comment by abhinavsri360** [comment]
@todcru
Make sure to pull before push.

File: src/components/json/worldData.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import abhinavsri360 from "../../photos/abhinavsri360.jpg";
 import AravindVNair99 from "../../photos/AravindVNair99.jpg";
 import srilaasya from "../../photos/srilaasya.jpg";
 import HarshJain from '../../photos/HarshJain.jpeg'
+import Aswin from "../../photos/Aswin.JPG";
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:6]:
```suggestion
import Aswin from "../../photos/Aswin.jpg";
```

**Pull request closed without merging**